packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/use-flow.md

```diff
@@ -0,0 +1,146 @@
+The following sections outline the requests required to implement the MFA OOB flow using direct calls to the Okta OIDC & OAuth 2.0 API.
+
+### Initial request for tokens
+
+Before you can begin this flow, collect the username and password from the user in a manner of your choosing. Then, make an API call to the Okta [authorization server](/docs/concepts/auth-servers/) `/token` endpoint using the Resource Owner Password grant type. If you're using the [default custom authorization server](/docs/concepts/auth-servers/#default-custom-authorization-server), then your request would look something like this:
+
+```bash
+curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/token \
```

Comment:
I actually think the most common usecase will be the org authorzation server so let's remove the reference to default custom AS and update URL to `https://${yourOktaDomain}/oauth2/v1/token`